src/wallet/wallet.ts

```diff
@@ -135,6 +145,39 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
    **/
   clearSensitiveData() {
     this.seed = null;
+    this.authXpriv = null;
+  }
+
+  /**
+   * Get auth xpubkey in the 280' purpose path from seed
+   *
+   * @param {String} seed 24 words
+   * @param {Object} options Options with passphrase and networkName
+   *
+   * @return {String} auth xpubkey
+   * @memberof HathorWalletServiceWallet
+   * @inner
+   */
+  static getAuthXPubKeyFromSeed(seed: string, options: { passphrase?: string, networkName?: string } = {}): string {
+    const methodOptions = Object.assign({passphrase: '', networkName: 'mainnet', accountDerivationIndex: '0\''}, options);
```

Comment:
Why do you need `accountDerivationIndex`?

Reply:
We don't need it

Removed in 504cf8c44a754aab33c85a3f99791de9e080afa3